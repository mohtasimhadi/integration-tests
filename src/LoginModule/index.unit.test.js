import React from 'react';
import { render } from '@testing-library/react';
import { LoginModule } from './';

describe('LoginModule', () => {
  it('matches snapshot', () => {
    const { container } = render(<LoginModule />);
    expect(container.cloneNode(true)).toMatchSnapshot();
  });
});
