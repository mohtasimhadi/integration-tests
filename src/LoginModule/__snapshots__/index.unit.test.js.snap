// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginModule matches snapshot 1`] = `
<div>
  <form
    autocomplete="off"
    novalidate=""
  >
    <div
      class="MuiBox-root MuiBox-root-2"
    >
      <div
        class="MuiFormControl-root MuiTextField-root"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
          data-shrink="false"
          for="email"
          id="email-label"
        >
          Email
        </label>
        <div
          class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiOutlinedInput-input"
            id="email"
            name="email"
            type="text"
            value=""
          />
          <fieldset
            aria-hidden="true"
            class="PrivateNotchedOutline-root-61 MuiOutlinedInput-notchedOutline"
          >
            <legend
              class="PrivateNotchedOutline-legendLabelled-63"
            >
              <span>
                Email
              </span>
            </legend>
          </fieldset>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-65"
    >
      <div
        class="MuiFormControl-root MuiTextField-root"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
          data-shrink="false"
          for="password"
          id="password-label"
        >
          Password
        </label>
        <div
          class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiOutlinedInput-input"
            id="password"
            name="password"
            type="password"
            value=""
          />
          <fieldset
            aria-hidden="true"
            class="PrivateNotchedOutline-root-61 MuiOutlinedInput-notchedOutline"
          >
            <legend
              class="PrivateNotchedOutline-legendLabelled-63"
            >
              <span>
                Password
              </span>
            </legend>
          </fieldset>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-66"
    >
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
        tabindex="0"
        type="submit"
      >
        <span
          class="MuiButton-label"
        >
          Submit
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
  </form>
</div>
`;
